scripts/check_doc_links.py

```diff
@@ -49,6 +49,8 @@
 HTTP_SKIPS = [
     "http://www.fipa.org/repository/ips.php3",
     "http://host.docker.internal:8545",
+    "http://host.docker.internal:8545",  # internal (ERR_NAME_NOT_RESOLVED)
+    "http://host.docker.internal:8545}",  # regex issue
```

Comment:
Why repeated 3 times?

Reply:
There are two internal docker links in our docs: the ones set as environment variable (written inside brackets) and the "normal ones". There's two problems with those 
* They are HTTP links
* The regex gets the } as part of the link in the ones inside brackets
So I needed to add both of them to both HTTP and HTTPS skips.

Reply:
The URL on lines 51 and 52 are the same

Reply:
Fixed